src/extensions/agent_modules/shared/agentinfo_utils/src/adu_agentinfo_utils.c

```diff
@@ -92,20 +92,18 @@ AgentInfoDataFromRetriableOperationContext(ADUC_Retriable_Operation_Context* ret
  */
 bool AgentInfoData_SetCorrelationId(ADUC_AgentInfo_Request_Operation_Data* agentInfoData, const char* correlationId)
 {
-    int ret = -1;
     if (agentInfoData == NULL || correlationId == NULL)
     {
         return false;
     }
 
-    ret = ADUC_Safe_StrCopyN(
+    if (!ADUC_Safe_StrCopyN(
```

Comment:
would ordering this api to match [StringCchCopyNA](https://learn.microsoft.com/en-us/windows/win32/api/strsafe/nf-strsafe-stringcchcopyna) be worthwhile?  I like that ordering better as it keeps lengths next to the args they reference.

Reply:
That makes sense to put the relevant counts adjacent. Will send that as a separate PR.

Reply:
Opened this to track: [Bug 48174247](https://microsoft.visualstudio.com/OS/_workitems/edit/48174247): [Gen2]Align ADUC_Safe_StrCopyN arg order with StringCchCopyNA

Reply:
@JeffMill PR to address is here: https://github.com/Azure/iot-hub-device-update/pull/590